src/krux/camera.py

```diff
@@ -87,8 +87,8 @@ def initialize_sensor(self, mode=QR_SCAN_MODE):
         """Initializes the camera"""
         sensor.reset(freq=18200000)
         self.cam_id = sensor.get_id()
-        if board.config["type"] == "cube" or (
-            board.config["type"] in ["yahboom", "wonder_mv"]
```

Comment:
Yahboom and WonderMV were excluded here.

Reply:
yes see the lines below :stuck_out_tongue_closed_eyes:  only wonder_mv and yahboom CAN_FLIP_ORIENTATION